### PR TITLE
Update cookies.md for clear regex explanation 

### PR DESCRIPTION
**Pull request by conorbauer23**
Closes #

## Summary

**[Working with Cookies on Pantheon](https://pantheon.io/docs/cookies)** - Current cookies page lists a regex, which is sort of clear to developers, but not clear to non-developers. Let's make it easy by clearly stating it's a regex that is required, and supplying a path to check for regex compatibility.

**Release**:
- [x] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by conorbauer23** [comment]
@CdrMarks Anything you want to add to this one? I think we discussed this recently.

File: source/content/cookies.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
 
 ## Cache-Varying Cookies
 
-Respond to a request with cached content depending on the presence and value of a particular cookie. It's important to note that in order for the response to be cached by Pantheon's Edge, the cookie name must match `STYXKEY[a-zA-Z0-9_-]+`.
+Respond to a request with cached content depending on the presence and value of a particular cookie. It's important to note that in order for the response to be cached by Pantheon's Edge, the cookie name must match the following regular expression format: `STYXKEY[a-zA-Z0-9_-]+`. To make sure the cookie in question matches `STYXKEY[a-zA-Z0-9_-]+`, test your cookie with [Regex 101.](https://regex101.com/)
```
Comment (CdrMarks) [inline source/content/cookies.md:60]: I would like to see some changes, such as this, made to this PR:
```suggestion
Respond to a request with cached content depending on the presence and value of a particular cookie. It's important to note that in order for the response to be cached by Pantheon's Edge, the cookie name must follow these naming rules:
* Must begin with the 7 capital letters `STYXKEY`
* Must have one or more of the following:
	* letters `a-z` and `A-Z`
	* numbers `0-9`
	* underscore `_`
	* hyphen `-`

If you prefer a regular expression, the name must match the following format: `STYXKEY[a-zA-Z0-9_-]+`. Test your cookie with [Regex 101.](https://regex101.com/) 
```

**Review by CdrMarks — CHANGES_REQUESTED** [review]
I've made a suggestion.  If you like it, there is a link to accept the suggestion.

**Comment by conorbauer23** [comment]
Suggestion accepted! Thanks @CdrMarks

**Review by CdrMarks — APPROVED** [review]

**Comment by CdrMarks** [comment]
@whitneymeredith was there something wrong with this PR.  I'm confused why this one was closed and 7500 was merged instead?

**Comment by whitneymeredith** [comment]
@CdrMarks I received an error (error: pathspec did not find a match) when I tried to check out Conor's branch on my local.

**Pull request closed without merging**